docs/notebooks/expected_improvement.pct.py

```diff
@@ -201,6 +201,54 @@ def build_model(data):
     idx_best=arg_min_idx,
 )
 
+# %% [markdown]
+# ## Batch-sequential strategy
+#
+# Sometimes it is practically convenient to query several points at a time. We can do this in `trieste` using a `BatchAcquisitionRule` and a `BatchAcquisitionFunctionBuilder`, that together recommend a number of query points `num_query_points` (instead of one as previously). The optimizer then queries the observer at all these points simultaneously.
+# Here we use the `BatchMonteCarloExpectedImprovement` function. Note that this acquisition function is computed using a Monte-Carlo method (so it requires a `sample_size`), but with a reparametrisation trick, which makes it deterministic.
+
+# %%
+qei = trieste.acquisition.BatchMonteCarloExpectedImprovement(sample_size=1000)
+batch_rule = trieste.acquisition.rule.BatchAcquisitionRule(num_query_points=3, builder=qei.using(OBJECTIVE))
+
+model = build_model(initial_data[OBJECTIVE])
+batch_result = bo.optimize(5, initial_data, model, acquisition_rule=batch_rule)
+batch_dataset = batch_result.try_get_final_datasets()[OBJECTIVE]
+batch_query_points = batch_dataset.query_points.numpy()
+batch_observations = batch_dataset.observations.numpy()
```

Comment:
i've approved, but perhaps move this down a cell, so users can see this `numpy`-ing is plotting specific and they don't need to do it if they don't want?

Reply:
entirely up to you